geo_activity_playground/core/activity_parsers.py

```diff
@@ -120,13 +123,26 @@ def read_fit_activity(path: pathlib.Path, open) -> pd.DataFrame:
                             row["altitude"] = fields["altitude"]
                         if "enhanced_altitude" in fields:
                             row["altitude"] = fields["enhanced_altitude"]
+                        if "grade" in fields:
+                            row["grade"] = fields["grade"]
                         if "speed" in fields:
                             row["speed"] = fields["speed"]
                         if "enhanced_speed" in fields:
                             row["speed"] = fields["enhanced_speed"]
+                        if "temperature" in fields:
+                            row["temperature"] = fields["temperature"]
+                        if "gps_accuracy" in fields:
+                            row["gps_accuracy"] = fields["gps_accuracy"]
                         rows.append(row)
-
-    return pd.DataFrame(rows)
+                    elif "wkt_name" in fields and "sport" in fields and "sub_sport" in fields:
+                        metadata["wkt_name"] = fields["wkt_name"]
+                        metadata["sport"] = (fields["sport"], fields["sub_sport"])
```

Comment:
How does _sport_ and _sub sport_ differ from the activity _kind_ (ride, walk, hike)? Can you give examples?

Reply:
Ah, so it is just something “cycling” and “generic”. I've added that into the `kind` field.